scripts/parse_stress_results.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+yaml_file="$1"
```

Comment:
I could be wrong, but merging these all into a single json (and merging new results w/ old) is probably gonna be enough of a pain you wanna do it in python tbh.

Reply:
I think maybe we should just make it output in the criterion output format, then when we add Rust specific benchmarks it can all use the same output.